### PR TITLE
Minor doc changes based on the recent PRs

### PR DESCRIPTION
**Pull request by vkurup**
Namely #250 and #251

### CODE REVIEW

File: docs/dev/provisioning.rst [hunk 1]
```diff
@@ -58,19 +58,17 @@ Pillar Setup
 
 Before your project can be deployed to a server, the code needs to be
 accessible in a git repository. Once that is done you should update
-``conf/pillar/<environment>.sls`` to set the repo and branch for the environment.
+``conf/pillar/project.sls`` to set the repo and branch for the environment.
```
Comment (dpoirier) [inline docs/dev/provisioning.rst]: I guess this is where we should talk about setting the branch.

**Comment by vkurup** [comment]
@dpoirier good catch. I added some docs, and perhaps in the 'foolish-consistency' department, I swapped the order of the variables in staging.sls to match production.sls

File: docs/dev/provisioning.rst [hunk 2]
```diff
@@ -220,9 +226,9 @@ Setup Checklist
 
 To summarize the steps above, you can use the following checklist
 
-- ``repo`` is set in ``conf/pillar/<environment>.sls``
-- Developer user names and SSH keys have been added to ``conf/pillar/devs.sls``
+- ``repo`` is set in ``conf/pillar/project.sls``
 - Project name has been set in ``conf/pillar/project.sls``
+- Developer user names and SSH keys have been added to ``conf/pillar/devs.sls``
 - Environment domain name has been set in ``conf/pillar/<environment>.sls``
 - Environment secrets including the deploy key have been set in ``conf/pillar/<environment>.sls``
```
Comment (dpoirier) [inline docs/dev/provisioning.rst]: Maybe add "branch name has been..." to this list?

**Comment by dpoirier** [comment]
One more suggestion, :shipit: either way.

**Comment by vkurup** [comment]
@dpoirier updated. Want to take another quick peek? You still have a 'requested changes' block on the PR.

**Comment by dpoirier** [comment]
Github won't show me the diffs right now, but I trust you :-)   :shipit:

**Pull request merged**